python/pip.bzl

```diff
@@ -34,6 +34,14 @@ def _pip_import_impl(repository_ctx):
   if result.return_code:
     fail("pip_import failed: %s (%s)" % (result.stdout, result.stderr))
 
+def _pip2_import_impl(repository_ctx):
```

Comment:
Not sure, but maybe there should be three options? `pip_import` which uses system `python`, `pip2_import` which explicitly uses `python2`, and `pip3_import` which explicitly uses `python3`?